docs/example-select.js

```diff
@@ -0,0 +1,19 @@
+function switchTab(event) {
+    let tab = event.target
+
+    let lang = tab.dataset.lang
+
+    let examples = tab.closest(".language-examples")
+
+    examples.querySelector('.tabs .selected').classList.remove("selected")
+    tab.classList.add("selected")
+
+    examples.querySelector('.examples .selected').classList.remove("selected")
+    examples.querySelector(`.examples > [data-lang="${lang}"]`).classList.add("selected")
+}
+
+document.addEventListener("DOMContentLoaded", function(event) {
+    document.querySelectorAll(".tabs > div").forEach(tab => {
```

Comment:
super nit: probably would want to give the children a `tab` class to make this selector less brittle but i imagine the markup for this won't change very frequently anyway. below you could then also switch to `> .tab` `> .tab:first-child` etc

Reply:
Ah yeah, I guess I'm being lazy.